gef.py

```diff
@@ -6345,14 +6345,20 @@ def print(self) -> None:
         for chunk_flag, chunk_summary in self.flag_distribution.items():
             gef_print("{:<15s}\t{:<10d}\t{:<d}".format(chunk_flag, chunk_summary.count, chunk_summary.total_bytes))
 
+class GlibcHeapWalkContext:
```

Comment:
`namedtuple` is probably more appropriate.

Reply:
Though I see you're trying to not have to pick out the args in the call to inialize. That might be OK.